tests/gold_tests/pluginTest/ja3_fingerprint/ja3_fingerprint.replay.yaml

```diff
@@ -0,0 +1,126 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Test Notes:
+# This test makes the server reply slowly over multiple origin connections,
+# allowing client-side connections to build up. This allows us to test our
+# proxy.config.net.per_client.max_connections_in feature. This test assumes that
+# proxy.config.net.per_client.max_connections_in is configured to 3 and assumes
+# the fourth connection will be aborted by ATS.
```

Comment:
This testing is exercising max_connections as well as ja3?

Reply:
Nah. It's copied and pasted from the max connections test. :) 

Good catch. I'll update the test. I've also realized that the plugin doesn't really play well as both a remap and global plugin. It's not designed to do both. It's currently trying to create two log files. I'll update the test and the README to convey this, and fix the comment issue you noted.